make verify broken: scripts/generate_mocks.sh returns error

./scripts/generate_mocks.sh
prog.go:14:2: no required module provides package github.com/rancher/elemental-operator/pkg/register: go.mod file not found in current directory or any parent directory; see 'go help modules'
prog.go:12:2: no required module provides package go.uber.org/mock/mockgen/model: go.mod file not found in current directory or any parent directory; see 'go help modules'
2023/09/19 18:09:43 Loading input failed: exit status 1
prog.go:14:2: no required module provides package github.com/rancher/elemental-operator/pkg/register: go.mod file not found in current directory or any parent directory; see 'go help modules'
prog.go:12:2: no required module provides package go.uber.org/mock/mockgen/model: go.mod file not found in current directory or any parent directory; see 'go help modules'
2023/09/19 18:09:49 Loading input failed: exit status 1
prog.go:14:2: no required module provides package github.com/rancher/elemental-operator/pkg/install: go.mod file not found in current directory or any parent directory; see 'go help modules'
prog.go:12:2: no required module provides package go.uber.org/mock/mockgen/model: go.mod file not found in current directory or any parent directory; see 'go help modules'
2023/09/19 18:09:55 Loading input failed: exit status 1
prog.go:14:2: no required module provides package github.com/rancher/elemental-operator/pkg/elementalcli: go.mod file not found in current directory or any parent directory; see 'go help modules'
prog.go:12:2: no required module provides package go.uber.org/mock/mockgen/model: go.mod file not found in current directory or any parent directory; see 'go help modules'
2023/09/19 18:09:59 Loading input failed: exit status 1

--- a/scripts/generate_mocks.sh
+++ b/scripts/generate_mocks.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-go install go.uber.org/mock/mockgen@latest
+go install go.uber.org/mock/mockgen@v0.2.0
 
 # Always create mock files into a "mocks" subfolder to be ignored in test coverage.
 # See codecov.yml for more info 

